src/modeler/AutoRest.Swagger/Model/ValidationUtilities.cs

```diff
@@ -0,0 +1,32 @@
+﻿using AutoRest.Core.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AutoRest.Swagger.Model.Utilities
+{
+    public static class ValidationUtilities
+    {
+        private static readonly string XmsPageable = "x-ms-pageable";
+
+        // determine if the 
+        public static bool IsXmsPageableOrArrayResponseOperation(Operation op, ServiceDefinition entity)
+        {
+            if (op.Extensions.GetValue<object>(XmsPageable) != null) return true;
+
+            if (!(op.Responses?.ContainsKey("200")??false)) return false;
```

Comment:
CSharp-y question here,  if we don't do ```(op.Responses?.ContainsKey("200")??false``` here, what would be the default boolean value of this expression in case Responses object is null